sonaranalyzer-dotnet/its/expected/Automapper/AutoMapper-025B9F93-7953-454B-AF29-E6A55B2AA58F-S4056.json

```diff
@@ -147,32 +56,6 @@
 "id":  "S4056",
 "message":  "Use the overload that takes a 'CultureInfo' or 'IFormatProvider' parameter.",
 "location":  {
-"uri":  "sources\Automapper\src\AutoMapper\Mappers\TypeConverterMapper.cs",
```

Comment:
`ConvertTo(object value, Type destinationType);` vs `ConvertTo(ITypeDescriptorContext context, CultureInfo culture, object value, Type destinationType);` - not compatible